LanguageMerger/LanguageFiles/tfg/en_us/Quests/tfg_tips.json

```
@@ -522,71 +522,69 @@
     "quests.tfg_tips.warfare.subtitle": "FIRE ALL CANNONS!",
     "quests.tfg_tips.warfare.desc": "You can outfit your &bSloop&r with firepower via the &dCannon&r and cannon balls, in case you want to blow something to smithereens.\nTo fire a cannon, you'll need to load it with some &aGunpowder, Unrefined Paper&r and your &aCannon Ball&r. Then light the wick with &cFlint and Steel.&r",
 
-    "quests.tfg_tips.kayak_beginning.title": "Boat 1: Kayak",
-    "quests.tfg_tips.kayak_beginning.subtitle": "The only portable boat",
-    "quests.tfg_tips.kayak_beginning.desc": "The kayak is a relatively simple boat to craft, while only one passenger can fit in it it makes up for it by being the only boat you can carry around with you.",
-
-    "quests.tfg_tips.kayak_materials.title": "Materials for a Kayak",
+    "quests.tfg_tips.kayak_materials.title": "The Kayak",
     "quests.tfg_tips.kayak_materials.subtitle": "Get yourself that Kayak.",
-    "quests.tfg_tips.kayak_materials.desc": "The Kayak is probably the second most simple boat you can craft, you'll require some form of string, some lumber, and Large Waterproof Hides. The Waterproof Hides can be obtained utilizing Waxes.",
+    "quests.tfg_tips.kayak_materials.desc": "The &dKayak&r is the second most simple boat you can craft, requiring &bWaxes&r for the &aWaterproof Hides&r. It is also the &conly&r boat that can be picked up as an item. Despite having a &dsingle passenger&r slot, it is the &cfastest boat&r in the game!",
```

have you tried a sloop in the ocean when the wind is blowing the right way? it is REALLY fast

---

Oh I only built it and not really used it.
```suggestion
    "quests.tfg_tips.kayak_materials.desc": "The &dKayak&r is the second most simple boat you can craft, requiring &bWaxes&r for the &aWaterproof Hides&r. It is also the &conly&r boat that can be picked up as an item. Despite having a &dsingle passenger&r slot, it is the fastest boat in the game until the steel age &dSloop!&r",
```